libraries/AP_Math/location.cpp

```diff
@@ -46,6 +46,30 @@ float longitude_scale(const struct Location &loc)
 #endif
 }
 
+// Added for use by SoaringController
+// return distance to north
+float get_offset_north(const struct Location &loc1, const struct Location &loc2)
+{
+    if (loc1.lat == 0 || loc1.lng == 0)
```

Comment:
Thanks, I reformatted the code (here and elsewhere) per the guidelines you mentioned.